docs/development.rst

```diff
@@ -93,58 +90,90 @@ To run migrations, you do::
 It'll perform any migrations that haven't been performed for all apps.
 
 
-Setting up an app for migrations
```

Comment:
It should be the same as the instructions below `./manage.py makemigrations <app>` so I felt that was enough. But it could be more explicit I guess.